spec/rollbar/plugins/basic_socket_spec.rb

```diff
@@ -14,6 +14,10 @@
 describe 'basic_socket plugin' do
   subject { Rollbar.plugins.get('basic_socket') }
 
+  before(:all) do
+    Rollbar.plugins.get('basic_socket').unload!
+  end
+
   after(:each) do
     subject.unload!
   end
```

Comment:
I'd be OK removing this `after` block, as each example now performs unload. I'm also OK keeping it.